debian/packages.d/systemd.customise.add/init.d/20confd.sh

```diff
@@ -0,0 +1,39 @@
+# Apply configurations from conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
```

Comment:
@hamishcoleman Works without this. Do we need?

Reply:
strange - I'm confused as to what is creating the device nodes in that case - this is running on a system with an empty /dev and before udev has hotplugged anything

Reply:
I was running the script manually so it wasn't tested at the correct state of OS startup

Reply:
ah, Doh!